packages/zowe-explorer/src/job/actions.ts

```diff
@@ -133,9 +151,9 @@ export async function getSpoolContentFromMainframe(node: IZoweJobTreeNode): Prom
                 node.getParent()
             );
             node = spoolNode;
-            await getSpoolContent(node.getProfile().name, spool, Date.now());
         }
     }
+    statusMsg.dispose();
```

Comment:
For context: I am manually disposing the status bar here in case polling was stopped in the middle of the request function. This way all of the polling status bar messages are cleared after polling has stopped.

I've noticed that sometimes VS Code will not dispose status bar items, so I definitely think we could benefit by having a linting rule for `.dispose()`.